app/views/govuk_component/govspeak_html_publication.raw.html.erb

```diff
@@ -2,7 +2,16 @@
   govspeak_locals = { content: content }
   govspeak_locals[:direction] = local_assigns.fetch(:direction) if local_assigns.include?(:direction)
   govspeak_locals[:rich_govspeak] = local_assigns.fetch(:rich_govspeak) if local_assigns.include?(:rich_govspeak)
+  sticky_footer_html = local_assigns.fetch(:sticky_footer_html) if local_assigns.include?(:sticky_footer_html)
 %>
-<div class="govuk-govspeak-html-publication">
+<div
+  class="govuk-govspeak-html-publication"
+  <%= "data-module=sticky-element-container" if sticky_footer_html %>
+>
   <%= render file: 'govuk_component/govspeak.raw', locals: govspeak_locals %>
+  <% if sticky_footer_html %>
```

Comment:
I'm not sure that the sticky element should be integrated into this component. I think I'd have preferred the JS module to wrap the component call here: https://github.com/alphagov/government-frontend/blob/b5b6e9ccd68b52e426a8471827d123e7fabe82ab/app/views/content_items/html_publication.html.erb#L27-L32